examples/misc/tickedstroke_demo.py

```diff
@@ -0,0 +1,83 @@
+"""
+============================
+TickedStroke Patheffect Demo
+============================
+
+"""
+
+import matplotlib.patches as patches
+from matplotlib.path import Path
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.patheffects as PathEffects
+
+# Direct to path
+plt.figure(figsize=(6, 6))
+ax1 = plt.subplot(111)
+path = Path.unit_circle()
+patch = patches.PathPatch(path, facecolor='none', lw=2, path_effects=[
+    PathEffects.withTickedStroke(angle=90, spacing=10, length=1, flip=True)])
+
+ax1.add_patch(patch)
+ax1.axis('equal')
+ax1.set_xlim(-2, 2)
+ax1.set_ylim(-2, 2)
+
+plt.show()
+
+# Lines and curves with plot and legend
```

Comment:
If you add 
~~~
#############################################################################
~~~
above you can make additional sections (see also [sphinx-gallery docs]https://sphinx-gallery.github.io/syntax.html#embed-rst-in-your-example-python-files). All previous plots are displayed before a new section starts. That way, you have each plot and code next to each other.

Alternatively, it may be feasible to shink everything into a single figure with (1, 3) subplots to make it more compact.

Reply:
I've tried this, I like the idea of grouping code & charts.